server/app/templates/admin.html

```diff
@@ -19,7 +19,7 @@
         <link href="/static/css/main.css" rel="stylesheet" type="text/css" />
 
         <link href="/static/vendor/angular-loading-bar/build/loading-bar.css" rel="stylesheet"></link>
-        <link href="/static/vendor/sweetalert/dist/sweetalert.css" rel="stylesheet"></link>
+        <link href="/static/vendor/sweetalert/dist/sweet-alert.css" rel="stylesheet"></link>
```

Comment:
I think this should be sweetalert.css - I updated the bower file so you may need to run bower install too see that.